ddtrace/contrib/openai/_endpoint_hooks.py

```diff
@@ -686,19 +688,19 @@ def _record_request(self, pin, integration, span, args, kwargs):
 
     def _record_response(self, pin, integration, span, args, kwargs, resp, error):
         resp = super()._record_response(pin, integration, span, args, kwargs, resp, error)
-        if not resp:
-            return
-        resp_to_tag = resp.model_dump() if hasattr(resp, "model_dump") else resp
-        if isinstance(resp_to_tag, str):
-            text = resp
-        elif isinstance(resp_to_tag, dict):
-            text = resp_to_tag.get("text", "")
-            if "segments" in resp_to_tag:
-                span.set_metric("openai.response.segments_count", len(resp_to_tag.get("segments")))
-        else:
-            text = ""
-        if integration.is_pc_sampled_span(span):
-            span.set_tag_str("openai.response.text", integration.trunc(text))
+        text = ""
+        if resp:
+            resp_to_tag = resp.model_dump() if hasattr(resp, "model_dump") else resp
+            if isinstance(resp_to_tag, str):
+                text = resp
+            elif isinstance(resp_to_tag, dict):
+                text = resp_to_tag.get("text", "")
+                if "segments" in resp_to_tag:
+                    span.set_metric("openai.response.segments_count", len(resp_to_tag.get("segments")))
+            else:
```

Comment:
super small nit: is this `else` needed now that you moved `text = ""` to the top?

Reply:
Thanks for the catch!